python/tests/fixtures/feature_descriptive_statistics_fixtures.json

```diff
@@ -0,0 +1,198 @@
+{
+  "get_fractional_feature_statistics": {
```

Comment:
I really think we should have distinct fixtures for numerical and categorical features